package.json

```diff
@@ -28,28 +28,29 @@
   "files": [
     "./build"
   ],
+  "type": "module",
```

Comment:
**Issue:** I'm not ready to make this an ES module.

I'd prefer to leave this as a CommonJS module for the time being.